spot-client/src/common/app-state/spot-tv/selectors.js

```diff
@@ -57,3 +57,14 @@ export function getRemoteSpotTVRoomName(state) {
 export function isConnectedToSpot(state) {
     return Boolean(state.spotTv.spotId);
 }
+
+/**
+ * A selector which returns whether or not the Spot-TV has volume control
+ * support.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {boolean}
+ */
+export function isVolumeControlSupported(state) {
```

Comment:
is that used anywhere ?

Reply:
Not right now. It's for the volume control work that should be landing this week.

Reply:
Can we land it with that work as first commit in the PR ?

Reply:
That’s fair. I’ll leave it I zoli.